src/components/programs/programheader.tsx

```diff
@@ -18,7 +18,7 @@ const ProgramHeader = ({
 }: HeaderProps) => {
   return (
     <div className="text-acm-gray-500 relative pb-10 text-2xl font-semibold">
-      <Image className="absolute" src={ellipse} alt="green" />
+      {ellipse && <Image className="absolute" src={ellipse} alt="green" />}
```

Comment:
```suggestion
      {ellipse && <Image className="absolute" src={ellipse} alt="ellipse" />}
```
not part of your issue but let's change this alt text to ellipse anyways